src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java

```diff
@@ -76,9 +96,14 @@ public Float validateFloat(final String setting, final float[] range, final bool
         if (paramMap.containsKey(setting)) {
             final String value = paramMap.remove(setting);
 
-            if (!value.isEmpty() && value.matches("-?\\d+(\\.\\d+)?")) {
-                final float fValue = Float.parseFloat(value);
-                if (fValue < range[0] && range[1] < fValue) {
+            if (!value.isEmpty()) {
+                final float fValue;
+                if (value.matches("-?\\d+\\.?(\\d+)?")) {
+                    fValue = Float.parseFloat(value);
+                } else {
+                    fValue = Float.NaN;
+                }
+                if (Float.isNaN(fValue) || (fValue < range[0] || range[1] < fValue)) {
```

Comment:
@KieranFrance did you intend to change this?

Reply:
I did, it was accidentally caught up in this commit.